scripts/wrapLatestCommitMsg.fsx

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System.IO
+open System
+
+#load "../src/FileConventions/Library.fs"
+
+#r "nuget: Fsdk, Version=0.6.0--date20230214-0422.git-1ea6f62"
+
+open Fsdk
+open Fsdk.Process
+
+let commitMsg =
+    Fsdk
+        .Process
+        .Execute(
+            {
+                Command = "git"
+                Arguments = "log -1 --format=%B"
+            },
+            Echo.Off
+        )
+        .UnwrapDefault()
+        .Trim()
+
+let header, maybeBody =
+    let newLineIndex = commitMsg.IndexOf Environment.NewLine
+
+    if newLineIndex > 0 then
+        commitMsg.Substring(0, newLineIndex).Trim(),
+        Some(commitMsg.Substring(newLineIndex).Trim())
+    else
+        commitMsg, None
+
+let maxCharsPerLine = 64
+
+let wrappedBody =
+    match maybeBody with
+    | Some body -> FileConventions.WrapText body maxCharsPerLine
+    | _ -> String.Empty
+
+let newCommitMsg =
+    header
+    + Environment.NewLine
+    + Environment.NewLine
+    + wrappedBody
+    + Environment.NewLine
+
+Fsdk
+    .Process
+    .Execute(
+        {
+            Command = "git"
+            Arguments = $"commit --amend -m \"{newCommitMsg}\""
```

Comment:
@realmarv btw should use a flag with a word, not with a single letter; and I asked if you tested it because I wonder what would happen with commit msgs that contain double quotes?

Reply:
triple quotes can be used for defining a raw string in git command but it's not working with Fsdk for some reason

Reply:
maybe it's just easier to get back to the temporary file option

Reply:
triple quotes is an F# syntax, you're dealing here with commandline related things, something entirely different; isn't it just enough to replace all double-quote chars with an escaped double-quote?

Reply:
Done